sigmf-spec.md

```diff
@@ -79,6 +79,10 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 JSON keywords are used as defined in [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
 
+Fields defined as "human-readable", a "string", or simply as "text" shall be
+treated as plaintext ASCII where whitespace is respected, unless otherwise
```

Comment:
@kpreid - You're right. ECMA-404 specifies `Unicode`, not a specific encoding. Good catch.

I'm good with your suggestion.